packages/redux/src/checkout/actions/factories/createCheckoutOrderContextFactory.ts

```diff
@@ -0,0 +1,65 @@
+import * as actionTypes from '../../actionTypes.js';
+import {
+  type CheckoutOrder,
+  type Config,
+  type PostCheckoutOrderContext,
+  type PostCheckoutOrderContextData,
+  type PostCheckoutOrderContextResponse,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import { normalize } from 'normalizr';
+import checkoutOrderContext from '../../../entities/schemas/checkoutOrderContext.js';
+import type { CreateCheckoutOrderContextAction } from '../../types/index.js';
+import type { Dispatch } from 'redux';
+
+/**
+ * Method responsible for creating a checkout order context.
+ *
+ * @param postCheckoutOrderContext - Post checkout order context client.
+ *
+ * @returns Thunk factory.
+ */
+const createCheckoutOrderContext =
+  (postCheckoutOrderContext: PostCheckoutOrderContext) =>
+  (
+    checkoutOrderId: CheckoutOrder['id'],
+    data: PostCheckoutOrderContextData,
+    config?: Config,
+  ) =>
+  async (
+    dispatch: Dispatch<CreateCheckoutOrderContextAction>,
+  ): Promise<PostCheckoutOrderContextResponse> => {
+    try {
+      dispatch({
+        type: actionTypes.CREATE_CHECKOUT_ORDER_CONTEXT_REQUEST,
+      });
+
+      const result = await postCheckoutOrderContext(
+        checkoutOrderId,
+        data,
+        config,
+      );
+      // The contextId is only accessible through the 'location' header.
+      const contextId: string =
+        result?.headers['location']?.split('contexts/')[1] || '';
```

Comment:
```suggestion
        result?.headers.location?.split('contexts/')[1] || '';
```

Nitpick (I guess it works the same 😅)

Reply:
done, can you check please?